defaults/k8s_local_kind_config.yaml

```diff
@@ -0,0 +1,33 @@
+instance_provision: 'on_demand'
+
+minikube_version: '0.11.0'
```

Comment:
Add comment and:
`minikube_version` -> `mini_k8s_version`

Reply:
Actually "minikube" is the name of project.
https://minikube.sigs.k8s.io/docs/start/

Reply:
@vponomaryov , you are right, but this is perfect case when we can avoid growing number of sct options by having general option that could be reused by multiple backends

Reply:
@vponomaryov `mini_k8s_version` is a general variable for storing local minikube versions for minikube and kind, we made this decision in order not to have many variables for all of the "mini" K8S backends